doc/cookbook/source/templates/page.html

```diff
@@ -20,6 +20,25 @@
 <script>
 jQuery(document).ready(function ($) {
     $('ul[id^=tabs]').tab();
+    $('.highlight').mouseup(function() {
+	    var timeout;
+	    var copyText = document.getSelection().toString();
+      if(copyText){
+        clearTimeout(timeout);
+        document.execCommand("copy");
+        $(this).tooltip("show");
+	    }    
+      var selected = this;
+      timeout = setTimeout(function() {
+      $(selected).tooltip("hide");
+      }, 1000);
+    });
+
+    $('.highlight').tooltip({
+	    trigger:'manual',
+      placement:'bottom',
```

Comment:
can the tooltip be placed relative to the cursor? I am thinking it is a bit arbitrary that it is in the middle

Reply:
Yes, but for that I need to wrap the text in one single element and then add tooltip to it,right now each letter has its own tag that's why I had to target the complete box. Do I need to look into sgexample.py for this?